pkg/kubelet/volumes.go

```diff
@@ -65,11 +63,13 @@ func (vh *volumeHost) NewWrapperMounter(volName string, spec volume.Spec, pod *a
 		spec.Volume.Name = wrapperVolumeName
 	}
 
-	b, err := vh.kubelet.newVolumeMounterFromPlugins(&spec, pod, opts)
-	if err == nil && b == nil {
-		return nil, errUnsupportedVolumeType
+	// newVolumeMounterFromPlugins can only return a valid mounter or an error
+	mounter, err := vh.kubelet.newVolumeMounterFromPlugins(&spec, pod, opts)
+	if err != nil {
+		return nil, err
 	}
-	return b, nil
+
+	return mounter, nil
```

Comment:
you don't need to check the return values here if you are just returning them. Just `return vh.kubelet.newVolumeMounterFromPlugins(&spec, pod, opts)` should do it